CHANGELOG.md

```diff
@@ -1,4 +1,41 @@
-2.27.0 Release notes (2019-5-10)
+X.Y.Z Release notes
```

Comment:
```suggestion
2.27.0 Release notes
```

Reply:
No, `npm run set-version 2.27.0` does that.